### PR TITLE
Tigon SQL changed interface setup

### PR DESCRIPTION
**Pull request by achalpandey**
Changed interface setup and added query rewriting to comply with Tigon SQL compiler expectations
- Now each defined schema has an associated interface named `"<schemaName>"` ~~\+ "s"~~ defined in file `<hostname>.ifq`
- ~~Each interface definition has a new field `<SubStream>` in `ifres.xml`~~
- ~~The Tigon SQL queries are re-written, such that each occurrence of `" " +  "<schemaName>" + " "` is replaced by `" [" + "<schemaName>" + "]." + "<schemaName>" + " "` to create the corresponding gsql file.~~
- The user is now expected to specify the interface as well as the schema while writing a Tigon SQL query

### CODE REVIEW

**Comment by ted-johnson** [comment]
I would very strongly advise against invisibly rewriting the user's query.
This solution is very messy and does nor scale.
You will very soon regret this decision if you commit to it.

**Comment by gokulavasan** [comment]
@ted-johnson Thanks for the input Ted. We are going to revise this PR based on our conversation.

File: tigon-sql/src/main/java/co/cask/tigon/sql/internal/StreamConfigGenerator.java [hunk 1]
```diff
@@ -79,8 +79,19 @@ private String createIfresXML(Set<String> inputNames) {
     return stringBuilder.toString();
   }
 
-  private String createLocalHostIfq(String hostname) {
-    return "default : Contains[InterfaceType, GDAT]";
+  /**
+   * This function generates the content for *.ifq file. It generates one interface for each schema.
+   * The interface name is auto-generated and is the same as the schema name.
+   * @param schemaMap Map containing input schemas
+   * @return
```
Comment (gokulavasan) [inline tigon-sql/src/main/java/co/cask/tigon/sql/internal/StreamConfigGenerator.java]: Returns?

Comment (gokulavasan) [inline tigon-sql/src/main/java/co/cask/tigon/sql/internal/StreamConfigGenerator.java]: Add a sample format of localhost.ifq (last line does not end with ';') ?

**Comment by gokulavasan** [comment]
LGTM :+1: Do the new tar balls contain fixes in PR https://github.com/caskdata/tigon/pull/86 ?

**Comment by achalpandey** [comment]
Updated tarballs just to be sure.

**Pull request merged**